test/test_load_simulation.jl

```diff
@@ -1,4 +1,105 @@
-include("../Simulations_preload.jl");
+using Pkg
+using Revise
+using PowerSystems
+const PSY = PowerSystems
+using Ipopt
+using GLPK
+using Dates
+using DataFrames
+using ParameterJuMP
+using JuMP
+using Test
+const PJ = ParameterJuMP
+import PowerModels
+const PM = PowerModels
+ipopt_optimizer = with_optimizer(Ipopt.Optimizer, print_level = 4)
+GLPK_optimizer = with_optimizer(GLPK.Optimizer, msg_lev = GLPK.MSG_ALL)
+using PowerSimulations
+const PSI = PowerSimulations
+using Random
+Random.seed!(232)
```

Comment:
All these include are made in `runtests.jl` you dont need to add them here, please remove these